--- multisync-0.83/plugins/evolution2_sync/configure.in	2005-10-07 01:22:29.076093336 +0200
+++ multisync-0.83-patrys/plugins/evolution2_sync/configure.in	2005-10-07 01:30:02.005237640 +0200
@@ -11,7 +11,7 @@
 AC_PROG_LIBTOOL
 AC_PROG_YACC
 
-pkg_modules="gtk+-2.0 libgnomeui-2.0 libbonoboui-2.0 glib-2.0 libbonobo-2.0 evolution-data-server-1.0 libebook-1.0 libecal-1.0 libedata-book-1.0 libedata-cal-1.0 libedataserver-1.0"
+pkg_modules="gtk+-2.0 libgnomeui-2.0 libbonoboui-2.0 glib-2.0 libbonobo-2.0 evolution-data-server-1.2 libebook-1.2 libecal-1.2 libedata-book-1.2 libedata-cal-1.2 libedataserver-1.2"
 PKG_CHECK_MODULES(PACKAGE, [$pkg_modules])
 AC_SUBST(PACKAGE_CFLAGS)
 AC_SUBST(PACKAGE_LIBS)
--- multisync-0.83/plugins/evolution2_sync/src/Makefile.am~	2006-01-18 19:11:46.000000000 +0100
+++ multisync-0.83/plugins/evolution2_sync/src/Makefile.am	2006-01-18 19:32:04.000000000 +0100
@@ -22,7 +22,7 @@
 	callbacks.c
 
 
-libevolution2_sync_la_LDFLAGS = @EVO_LDFLAGS@ -avoid-version -export-dynamic -module
-libevolution2_sync_la_LIBADD = @PACKAGE_LIBS@ -lpthread @EVO_LIBS@ 
+libevolution2_sync_la_LDFLAGS = -avoid-version -export-dynamic -module
+libevolution2_sync_la_LIBADD = @PACKAGE_LIBS@ -lpthread
 
 
