diff -urN multisync-0.82.org/plugins/irmc_sync/configure.in multisync-0.82/plugins/irmc_sync/configure.in
--- multisync-0.82.org/plugins/irmc_sync/configure.in	2004-08-07 01:11:44.091961288 +0000
+++ multisync-0.82/plugins/irmc_sync/configure.in	2004-08-07 01:12:38.108749488 +0000
@@ -22,8 +22,6 @@
 
 CPPFLAGS="${PACKAGE_CFLAGS}"
 AC_CHECK_LIB(bluetooth,baswap,BLUETOOTH=1,BLUETOOTH=0)
-SDP=0
-AC_CHECK_LIB(sdp,sdp_list_free,SDP=1,BLUETOOTH=0)
 AC_MSG_CHECKING(for correct SDP lib version)
 AC_TRY_COMPILE([
 #include <bluetooth/sdp.h>
@@ -31,7 +29,7 @@
 ],[sdp_record_t t;],AC_MSG_RESULT(yes),BLUETOOTH=0;AC_MSG_RESULT(no))
 AM_CONDITIONAL(BLUETOOTH,test "${BLUETOOTH}" = "1")
 if test "x${BLUETOOTH}" = "x0"; then
-  echo "Bluetooth is DISABLED. You must have bluez-libs and bluez-kernel >= 2.3 and bluez-sdp >= 1.0-pre1."
+  echo "Bluetooth is DISABLED. You must have bluez-libs and bluez-kernel >= 2.7."
 fi
 AC_DEFINE_UNQUOTED(HAVE_BLUETOOTH, ${BLUETOOTH},Defined if Bluetooth is enabled.)
 LIBS="$LIBS -lglib-2.0"
diff -urN multisync-0.82.org/plugins/irmc_sync/src/Makefile.am multisync-0.82/plugins/irmc_sync/src/Makefile.am
--- multisync-0.82.org/plugins/irmc_sync/src/Makefile.am	2004-08-07 01:11:44.093960984 +0000
+++ multisync-0.82/plugins/irmc_sync/src/Makefile.am	2004-08-07 01:28:44.959765792 +0000
@@ -21,7 +21,7 @@
 libirmc_bluetooth_la_SOURCES = \
 	irmc_bluetooth.c irmc_bluetooth.h
 
-libirmc_bluetooth_la_LIBADD = -lbluetooth -lsdp -lpthread
+libirmc_bluetooth_la_LIBADD = -lbluetooth -lpthread
 
 libirmc_sync_la_SOURCES = \
 	interface.c interface.h \
