diff -Nur multisync-0.82.orig/plugins/backup_plugin/configure.in multisync-0.82/plugins/backup_plugin/configure.in
--- multisync-0.82.orig/plugins/backup_plugin/configure.in	2004-04-12 06:22:56.000000000 +0200
+++ multisync-0.82/plugins/backup_plugin/configure.in	2004-06-03 22:15:20.000000000 +0200
@@ -16,15 +16,14 @@
 AC_PROG_LIBTOOL
 
 dnl Info for the RPM
-MULTISYNC_TOP="../.."
 AC_SUBST(VERSION)
 AC_SUBST(prefix)
-MULTISYNC_VERSION=`grep "#define VERSION" ${MULTISYNC_TOP}/config.h | sed -e 's/#define VERSION //g' | sed -e 's/\"//g'`
+MULTISYNC_VERSION=`grep "#define VERSION" ../../config.h | sed -e 's/#define VERSION //g' | sed -e 's/\"//g'`
 AC_SUBST(MULTISYNC_VERSION)
 
 AC_OUTPUT([
 Makefile
 src/Makefile
-${MULTISYNC_TOP}/specs/multisync-backup.spec
+../../specs/multisync-backup.spec
 ])
 
diff -Nur multisync-0.82.orig/plugins/evolution_sync/configure.in multisync-0.82/plugins/evolution_sync/configure.in
--- multisync-0.82.orig/plugins/evolution_sync/configure.in	2004-04-12 06:22:57.000000000 +0200
+++ multisync-0.82/plugins/evolution_sync/configure.in	2004-06-03 22:15:41.000000000 +0200
@@ -93,10 +93,9 @@
 AC_PROG_YACC
 
 dnl Info for the RPM
-MULTISYNC_TOP="../.."
 AC_SUBST(VERSION)
 AC_SUBST(prefix)
-MULTISYNC_VERSION=`grep "#define VERSION" ${MULTISYNC_TOP}/config.h | sed -e 's/#define VERSION //g' | sed -e 's/\"//g'`
+MULTISYNC_VERSION=`grep "#define VERSION" ../../config.h | sed -e 's/#define VERSION //g' | sed -e 's/\"//g'`
 AC_SUBST(MULTISYNC_VERSION)
 AC_SUBST(EVO_RPM_VER)
 
@@ -105,6 +104,6 @@
 src/Makefile
 intl/Makefile
 po/Makefile.in
-${MULTISYNC_TOP}/specs/multisync-evolution.spec
+../../specs/multisync-evolution.spec
 ])
 
diff -Nur multisync-0.82.orig/plugins/irmc_sync/configure.in multisync-0.82/plugins/irmc_sync/configure.in
--- multisync-0.82.orig/plugins/irmc_sync/configure.in	2004-04-12 06:22:58.000000000 +0200
+++ multisync-0.82/plugins/irmc_sync/configure.in	2004-06-03 22:15:56.000000000 +0200
@@ -63,10 +63,9 @@
 AC_PROG_LIBTOOL
 
 dnl Info for the RPM
-MULTISYNC_TOP="../.."
 AC_SUBST(VERSION)
 AC_SUBST(prefix)
-MULTISYNC_VERSION=`grep "#define VERSION" ${MULTISYNC_TOP}/config.h | sed -e 's/#define VERSION //g' | sed -e 's/\"//g'`
+MULTISYNC_VERSION=`grep "#define VERSION" ../../config.h | sed -e 's/#define VERSION //g' | sed -e 's/\"//g'`
 AC_SUBST(MULTISYNC_VERSION)
 
 
@@ -76,8 +75,8 @@
 src/bfb/Makefile
 intl/Makefile
 po/Makefile.in
-${MULTISYNC_TOP}/specs/multisync-irmc.spec
-${MULTISYNC_TOP}/specs/multisync-irmc-bluetooth.spec
+../../specs/multisync-irmc.spec
+../../specs/multisync-irmc-bluetooth.spec
 ])
 
 echo
diff -Nur multisync-0.82.orig/plugins/ldap_plugin/configure.in multisync-0.82/plugins/ldap_plugin/configure.in
--- multisync-0.82.orig/plugins/ldap_plugin/configure.in	2004-04-12 06:23:06.000000000 +0200
+++ multisync-0.82/plugins/ldap_plugin/configure.in	2004-06-03 22:16:11.000000000 +0200
@@ -20,14 +20,13 @@
 AC_CHECK_HEADER(sasl/sasl.h,, AC_MSG_ERROR(You must have the libsasl2-dev package installed))
 
 dnl Info for the RPM
-MULTISYNC_TOP="../.."
 AC_SUBST(VERSION)
 AC_SUBST(prefix)
-MULTISYNC_VERSION=`grep "#define VERSION" ${MULTISYNC_TOP}/config.h | sed -e 's/#define VERSION //g' | sed -e 's/\"//g'`
+MULTISYNC_VERSION=`grep "#define VERSION" ../../config.h | sed -e 's/#define VERSION //g' | sed -e 's/\"//g'`
 AC_SUBST(MULTISYNC_VERSION)
 
 AC_OUTPUT([
 Makefile
 src/Makefile
-${MULTISYNC_TOP}/specs/multisync-ldap.spec
+../../specs/multisync-ldap.spec
 ])
diff -Nur multisync-0.82.orig/plugins/opie_sync/configure.in multisync-0.82/plugins/opie_sync/configure.in
--- multisync-0.82.orig/plugins/opie_sync/configure.in	2004-04-12 06:23:06.000000000 +0200
+++ multisync-0.82/plugins/opie_sync/configure.in	2004-06-03 22:16:30.000000000 +0200
@@ -42,16 +42,15 @@
 AC_CHECK_LIB(crypto,MD5_Init,,AC_MSG_ERROR(You must have OpenSSL libcrypto installed.))
 
 dnl Info for the RPM
-MULTISYNC_TOP="../.."
 AC_SUBST(VERSION)
 AC_SUBST(prefix)
-MULTISYNC_VERSION=`grep "#define VERSION" ${MULTISYNC_TOP}/config.h | sed -e 's/#define VERSION //g' | sed -e 's/\"//g'`
+MULTISYNC_VERSION=`grep "#define VERSION" ../../config.h | sed -e 's/#define VERSION //g' | sed -e 's/\"//g'`
 AC_SUBST(MULTISYNC_VERSION)
 
 AC_OUTPUT([
 Makefile
 src/Makefile
 po/Makefile.in
-${MULTISYNC_TOP}/specs/multisync-opie.spec
+../../specs/multisync-opie.spec
 ])
 
diff -Nur multisync-0.82.orig/plugins/palm_sync/configure.in multisync-0.82/plugins/palm_sync/configure.in
--- multisync-0.82.orig/plugins/palm_sync/configure.in	2004-04-12 06:23:14.000000000 +0200
+++ multisync-0.82/plugins/palm_sync/configure.in	2004-06-03 22:16:42.000000000 +0200
@@ -20,14 +20,13 @@
 AC_CHECK_HEADER(pi-socket.h,, AC_MSG_ERROR(You must have the libpisock-dev package installed.))
 
 dnl Info for the RPM
-MULTISYNC_TOP="../.."
 AC_SUBST(VERSION)
 AC_SUBST(prefix)
-MULTISYNC_VERSION=`grep "#define VERSION" ${MULTISYNC_TOP}/config.h | sed -e 's/#define VERSION //g' | sed -e 's/\"//g'`
+MULTISYNC_VERSION=`grep "#define VERSION" ../../config.h | sed -e 's/#define VERSION //g' | sed -e 's/\"//g'`
 AC_SUBST(MULTISYNC_VERSION)
 
 AC_OUTPUT([
 Makefile
 src/Makefile
-${MULTISYNC_TOP}/specs/multisync-palm.spec
+../../specs/multisync-palm.spec
 ])
diff -Nur multisync-0.82.orig/plugins/syncml_plugin/configure.in multisync-0.82/plugins/syncml_plugin/configure.in
--- multisync-0.82.orig/plugins/syncml_plugin/configure.in	2004-04-12 06:23:16.000000000 +0200
+++ multisync-0.82/plugins/syncml_plugin/configure.in	2004-06-03 22:16:53.000000000 +0200
@@ -41,16 +41,15 @@
 AC_DEFINE_UNQUOTED(SYNCML_DEBUG,${DEBUG},Debug printouts)
 
 dnl Info for the RPM
-MULTISYNC_TOP="../.."
 AC_SUBST(VERSION)
 AC_SUBST(prefix)
-MULTISYNC_VERSION=`grep "#define VERSION" ${MULTISYNC_TOP}/config.h | sed -e 's/#define VERSION //g' | sed -e 's/\"//g'`
+MULTISYNC_VERSION=`grep "#define VERSION" ../../config.h | sed -e 's/#define VERSION //g' | sed -e 's/\"//g'`
 AC_SUBST(MULTISYNC_VERSION)
 AC_SUBST(EVO_RPM_VER)
 
 AC_OUTPUT([
 Makefile
 src/Makefile
-${MULTISYNC_TOP}/specs/multisync-syncml.spec
+../../specs/multisync-syncml.spec
 ])
 
